src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -84,7 +84,7 @@ namespace cryptonote
 // advance which version they will stop working with
 // Don't go over 32767 for any of these
 #define CORE_RPC_VERSION_MAJOR 2
-#define CORE_RPC_VERSION_MINOR 6
+#define CORE_RPC_VERSION_MINOR 8
```

Comment:
Any reason for skipping 7 ? Another PR somewhere ?

Reply:
As a matter of fact, #5549 uses 7. Not sure it's relevant, we'll rebase to whatever's the current version at the time anyway.

Reply:
Due to the https://github.com/monero-project/monero/pull/5582 PR. I guess it will be merged before this one